Portfolio.html

```diff
@@ -5,321 +5,396 @@
+              <li>
                 <a href="#">
-                  <img src="./images/twofingers.png" alt="two finger icon" />
-                </a>
+                  <img
+                    src="./Images/Mail.png"
+                    alt="Mail Icon"
+                    class="mail"
+                /></a>
               </li>
-              <li class="social-icon">
+              <li>
                 <a href="#">
-                  <img src="./images/twitter.png" alt="twitter icon" />
-                </a>
+                  <img
+                    src="./Images/Github.png"
+                    alt="Github Icon"
+                    class="github"
+                /></a>
               </li>
-              <li class="social-icon">
-                <a href="#">
-                  <img src="./images/Mail.png" alt="mail icon" />
-                </a>
+              <li>
+                <a href="#"
+                  ><img
+                    src="./Images/twofingers.png"
+                    alt="Other Icon"
+                /></a>
               </li>
             </ul>
           </div>
-          <button class="btn btn-resume">Get my resume</button>
+          <div class="more">
+            <button class="btn btn-resume" type="button">Get my resume</button>
+          </div>
         </div>
-
-        <!-- Skills acquired-->
-
-        <ul class="skillset">
-          <li class="languages">
-            <div class="lang-head">
+        <ul class="second">
+          <li class="language">
+            <div class="lang-section">
               <h2>Languages</h2>
-              <img src="./images/down-arrow.png" alt="Arrow Down" />
+              <img src="Images/down-arrow.png" alt="Arrow" />
             </div>
-            <ul class="language-list">
-              <li class="lang">
-                <img src="./images/js-Ellipse.png" alt="Javascript Image" />
-                <h3>Javascript</h3>
-              </li>
-              <li class="lang">
-                <img src="./images/html.png" alt="HTML Image" />
+            <ul class="languages">
+              <li class="lname">
+                <img src="Images/html.png" alt="html Icon" />
                 <h3>HTML</h3>
               </li>
-              <li class="lang">
-                <img src="./images/css-elipse.png" alt="CSS Image" />
+              <li class="desktop-version">
+                <img src="./images/js-Ellipse.png" alt="js-icon"/>
+                <hx>Javascript</hx>
+              </li>
+              <li class="lname">
+                <img src="Images/css-elipse.png" alt="Css Icon" />
                 <h3>CSS</h3>
               </li>
+              <li class="desktop-version">
+                <img src="./images/html.png" alt="html-tag"/>
+                <hx>HTML</hx>
+              </li>
+              <li class="lname">
+                <img src="Images/js-Ellipse.png" alt="Js Icon" />
+                <h3>Javascript</h3>
```

Comment:
- Please try to properly align-items on the webpage, as I noticed a breakage while resizing your webpage to a mobile view., and use the Figma design as a reference.

##### Figma design
![Screenshot from 2022-06-07 04-59-57](https://user-images.githubusercontent.com/58771507/172293261-2d2276c4-5077-49ca-9786-eb0b631010ae.png)


##### Your design 
![Screenshot from 2022-06-07 04-57-43](https://user-images.githubusercontent.com/58771507/172293079-082844e7-5612-4dab-a01a-0da8138dab24.png)